2020-08-19 18:29:18,407 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2020-08-19 18:29:18,407 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from D:\Python\lib\site-packages\instabot\bot\bot.py
2020-08-19 18:29:18,408 - instabot version: 0.117.0 (api_login) - DEBUG - Loading cookies
2020-08-19 18:29:18,408 - instabot version: 0.117.0 (api_login) - DEBUG - Loading uuids
2020-08-19 18:29:18,408 - instabot version: 0.117.0 (api_login) - INFO - Recovery from C:\Users\ellio\OneDrive\Documents\GitHub\memepage/config/freshly.squeeze_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=815992c6-d480-4fc1-bbfc-64ca4f7f5201
- uuid=14676004-ff81-40b9-92f8-d938c2be330c
- client_session_id=a7683b86-c450-457e-8729-039bd6b9814d
- device_id=android-d918cd0571e06043
2020-08-19 18:29:18,409 - instabot version: 0.117.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: False
2020-08-19 18:29:18,931 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/timeline/ returned response: <Response [403]>
2020-08-19 18:29:18,931 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "login_required", "error_title": "You\'ve Been Logged Out", "error_body": "Please log back in.", "logout_reason": 8, "status": "fail"}'
2020-08-19 18:29:18,931 - instabot version: 0.117.0 (api) - ERROR - Request returns 403 error!
2020-08-19 18:29:19,219 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/reels_tray/ returned response: <Response [403]>
2020-08-19 18:29:19,219 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "login_required", "error_title": "You\'ve Been Logged Out", "error_body": "Please log back in.", "logout_reason": 8, "status": "fail"}'
2020-08-19 18:29:19,219 - instabot version: 0.117.0 (api) - ERROR - Request returns 403 error!
2020-08-19 18:29:19,220 - instabot version: 0.117.0 (api) - INFO - Login flow failed, the cookie is broken. Relogin again.
2020-08-19 18:29:19,226 - instabot version: 0.117.0 (api_login) - DEBUG - Loading cookies
2020-08-19 18:29:19,227 - instabot version: 0.117.0 (api_login) - DEBUG - Loading uuids
2020-08-19 18:29:19,227 - instabot version: 0.117.0 (api_login) - INFO - Recovery from C:\Users\ellio\OneDrive\Documents\GitHub\memepage/config/freshly.squeeze_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=815992c6-d480-4fc1-bbfc-64ca4f7f5201
- uuid=14676004-ff81-40b9-92f8-d938c2be330c
- client_session_id=a7683b86-c450-457e-8729-039bd6b9814d
- device_id=android-d918cd0571e06043
2020-08-19 18:29:19,232 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2020-08-19 18:29:19,538 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2020-08-19 18:29:19,836 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2020-08-19 18:29:20,140 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2020-08-19 18:29:20,429 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2020-08-19 18:29:22,263 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2020-08-19 18:29:22,263 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "The password you entered is incorrect. Please try again.", "invalid_credentials": true, "error_title": "Incorrect password for freshly.squeeze", "buttons": [{"title": "Try Again", "action": "dismiss"}], "status": "fail", "error_type": "bad_password"}'
2020-08-19 18:29:22,263 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2020-08-19 18:29:22,264 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: The password you entered is incorrect. Please try again.
2020-08-19 18:29:22,264 - instabot version: 0.117.0 (api) - INFO - Error type: bad_password
2020-08-19 18:29:22,265 - instabot version: 0.117.0 (api) - ERROR - Failed to login go to instagram and change your password
2020-08-19 18:29:22,279 - instabot version: 0.117.0 (api) - INFO - Username or password is incorrect.
